code-reviews/recipes/Terraform.md

```diff
@@ -1,18 +1,23 @@
-# Terraform Code Reviews
+## Code Reviews
+### Style Guide
+- Adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html)
+- Ensure all used providers are versioned to prevent breaking changes in future. [docs on versioning providers](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
+- main.tf: This file usually contains provider configuration, backend configuration, imports to the modules to use and eventually some common resources that was not isolated into a specific file. It is absolutely possible to have the whole infrastructure defined in this file itself, suggest to split it into other *.tf files
+- README.md - Always useful to document what the Terraform project does
 
-## Style guide
+### Linters
+1. [tflint](https://github.com/terraform-linters/tflint) - As of now tflint doesn't have Azure specific rules, it can be used to inspect  general Terraform specific issues.  
 
-[CSE](../../CSE.md) developers adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html).
+### Useful extensions for VS Code and Visual Studio
 
-We also ensure all used providers are versioned to prevent breaking changes in future. [docs on versioning providers](https://www.terraform.io/docs/configuration/providers.html#provider-versions).
+The following VC Code extensions are widely used
 
-## Linting
+1. [Terraform extension](https://marketplace.visualstudio.com/items?itemName=mauve.terraform)
+2. [Azure Terraform extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureterraform)
 
-### tflint
+The Terraform extension also supports [tflint](https://github.com/terraform-linters/tflint/), tflint can be used independently and also within VS Code
```

Comment:
For consistency with other templates, add backticks around extension names

Reply:
Also, for consistency with other recipes, add these as separate ### titles and provide some explanation on what the extensions do.

Reply:
Updated based on the feedback.